### PR TITLE
Fix macos builds

### PR DESCRIPTION
**Pull request by m-kuhn**
🤞

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for da418861721a737f4d0ca74c81cf24858ea3fda6:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-e101133c8bcfdc556d0607d9a567a9281a3f9b48-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-e101133c8bcfdc556d0607d9a567a9281a3f9b48-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-e101133c8bcfdc556d0607d9a567a9281a3f9b48-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-e101133c8bcfdc556d0607d9a567a9281a3f9b48-x86-android.apk)

**Comment by m-kuhn** [comment]
https://github.com/jurplel/install-qt-action/issues/142

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f9e1d31af5f97e07a9a3f7e0900dc0cddb61c993:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7981b16786e8770c8ee49228240e87dabeb70aa8-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7981b16786e8770c8ee49228240e87dabeb70aa8-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7981b16786e8770c8ee49228240e87dabeb70aa8-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7981b16786e8770c8ee49228240e87dabeb70aa8-x86-android.apk)

File: .github/workflows/desktop.yml [hunk 1]
```diff
@@ -129,6 +133,7 @@ jobs:
           modules: 'qtcharts'
           target: ${{ steps.vars.outputs.QT_TARGET }}
           arch: ${{ steps.vars.outputs.QT_ARCH }}
+          py7zrversion: '==0.18.1'
```
Comment (nirvn) [inline .github/workflows/desktop.yml:136]:
```suggestion
          setup-python: false
          py7zrversion: '==0.18.1'
```

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for ab98fbdae322fec64d1e86e2ab84e38430d14593:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-b39b196e21081c9350399325890b568bf6c44ccf-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-b39b196e21081c9350399325890b568bf6c44ccf-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-b39b196e21081c9350399325890b568bf6c44ccf-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-b39b196e21081c9350399325890b568bf6c44ccf-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 457540570bbcf2eef3232853341fd12b2c9c39fa:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7f7b0c8139db7fd11f6d240515902f2b0fbf14e7-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7f7b0c8139db7fd11f6d240515902f2b0fbf14e7-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7f7b0c8139db7fd11f6d240515902f2b0fbf14e7-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2650-7f7b0c8139db7fd11f6d240515902f2b0fbf14e7-x86-android.apk)

**Comment by nirvn** [comment]
We've green again!

**Review by nirvn — APPROVED** [review]

**Pull request merged**